### DISCUSSION TITLE
Question about motif analysis vignette

### DISCUSSION THREAD

**Discussion by ahmedabbas811** [stuart-lab/signac#1522] [Q&A] [answered]
Hi all,
I have a question about motif analysis vignette. 
The analysis is about mouse brain. The Jaspar database is used which contain motifs for human and mouse. And the examples of the motifs in the end of the vignette (MA0497.1 for example) is MEF2C which is a human motif.

My question is, shouldn't we make any differentiation in the analysis so that it checks only for mouse motifs (of course I noticed that you use mouse genome in the analysis).

Thanks

**Comment by timoast** [comment, marked as answer]
We use the core vertebrate set of motifs in the vignette, you can certainly restrict to whatever set of motifs you think makes the most sense for your analysis. It is worth noting that the TF binding motifs are highly similar across vertebrates.
